spectrum/checks.py

```diff
@@ -881,7 +888,7 @@ def _resources_from(soup):
             srcset = media_source.get("srcset")
             if srcset:
                 resources.extend(_srcset_values(srcset))
-        return list(set(resources))
+        return sorted(list(set(resources)))
```

Comment:
slightly unrelated, however the Future objects are realised (elsewhere) in order so this may improve determinism a bit. For example, is it always the first iiif image request that fails and the rest pass? Right now it seems random.